x-pack/plugins/spaces/server/routes/api/external/copy_to_space.test.ts

```diff
@@ -365,58 +346,6 @@ describe('copy to space', () => {
       );
     });
 
-    it('does not allow namespace agnostic types to be copied (via "supportedTypes" property)', async () => {
```

Comment:
I think this is still something we want to prevent. It just so happens that we don't have any namespace-agnostic saved objects that are also importable/exportable, but if one was ever introduced, we'd want to prevent that from being copied between spaces. I don't think it will harm anything if it was attempted, but it will likely cause confusion for end users.

Reply:
As discussed offline: there are a few different ways we could skin this cat. Ultimately it seemed easiest/cleanest to make a change in the CTS code by filtering out any namespace-agnostic types between the "export" and "import" steps.

I made this change in 3538c2b.